admin/environment-management/cvms.md

```diff
@@ -21,6 +21,39 @@ supported.
 - The cluster must allow privileged containers and `hostPath` mounts. Read more
 about why this is still secure [here](#security).
 
+### Google Cloud Platform (GKE)
+
+- GKE Master version `>= 1.17`
+- `node-version >= 1.17` and `image-type = "UBUNTU"`
+
+#### Example with `gcloud`
+
+```bash
+gcloud beta container node-pools create "coder-pool" --cluster <cluster> \
+    --image-type "UBUNTU" \
+    --node-version "1.17.14-gke.1600" \
+    ...
```

Comment:
We should probably give the cluster create command rather than the pool create command. There's an example in `setup_k8s.sh` in the monorepo.